test/extended/util/test.go

```diff
@@ -431,6 +432,9 @@ var (
 
 			`Should be able to support the 1\.7 Sample API Server using the current Aggregator`, // down apiservices break other clients today https://bugzilla.redhat.com/show_bug.cgi?id=1623195
 		},
+		"[azure:Skipped]": {
```

Comment:
I think we should flip this `[Skipped:azure]`.

Reply:
+1 fixed with https://github.com/openshift/origin/compare/f3da840a3a95e884a362b837c68b6e49321fd6ac...a99a729e4239a3f3ea6c896906cd5f143f0aef96

@smarterclayton